tests/fltest_screen.fish

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env fish
+# -*-  mode:fish; tab-width:4  -*-
+
+function fltest_screen
+
+    echo "Context: STY var is set to 'fishline_test'"
```

Comment:
Sounds good, and all too familiar of a maintainer woe!  FWIW, I ran the test locally (ubuntu fish2.6 build) and it crashed out with a segfault, but I'm pretty sure that's on fish itself and not these tests.